ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/MovableBehaviour.java

```diff
@@ -0,0 +1,20 @@
+package com.github.migi_1.Context.model.entity;
+
+import com.jme3.math.Vector3f;
+
+/**
+ * class for handling the creation of a certain moving behaviour for an entity. 
+ * @author Damian
+ *
+ */
+public class MovableBehaviour {
+    
+    /**
+     * Creates a certain movable behaviour for an entity.
+     * @return Vector3f for the entity to move to.
+     */
+    public Vector3f getMoveVector() {
+        return new Vector3f(0, 0, 0);
+    }
```

Comment:
I would make this class abstract and turn this method into an abstract method getMoveVector. Then have a class StaticMoveBehaviour (or a similar name) that returns Vector3f(0, 0, 0).